giraffe/src/utils/normalizeConfig.ts

```diff
@@ -0,0 +1,22 @@
+import {Config, LayerConfig} from '../types'
+
+// TODO: move all layer defaults here, see: https://github.com/influxdata/giraffe/issues/447
+export const normalizeLayers = (layers: LayerConfig[]): LayerConfig[] =>
+  layers?.map(layerConfig => {
+    if (layerConfig.type === 'band') {
+      let {upperColumnName, lowerColumnName} = layerConfig
+      if (!upperColumnName || upperColumnName === layerConfig.mainColumnName) {
+        upperColumnName = ''
+      }
+      if (!lowerColumnName || lowerColumnName === layerConfig.upperColumnName) {
+        lowerColumnName = ''
+      }
+      return {...layerConfig, upperColumnName, lowerColumnName}
+    }
+    return layerConfig
+  }) || []
```

Comment:
if this got passed in as a default argument, it might make the code a little simpler, since we wouldn't need to check if `layers` exists before calling map on it

Reply:
Actually, I got a surprise. Making this change and then passing in `null` doesn't work.

I added a unit test to prove this. So, keep it as is, but with the requisite unit test to ensure